src/database/models.py

```diff
@@ -1879,6 +1881,9 @@ class Action(models.Model):
     is_approved = models.BooleanField(default=False, blank=True)
     # is_user_submitted = models.BooleanField(default=False, blank=True, null=True)
 
+    category= models.ForeignKey(CCCategory, blank=True, null=True, on_delete=models.SET_NULL, related_name="action_category")
```

Comment:
I don't think we need this, redundant.  If we change the carbon calculator action, these should reflect the category and subcategory of the new carbon calculator action.